packages/api/models/grantCycle.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
-const { Model } = require('sequelize');
+
+const { Model, Sequelize, DataTypes } = require('sequelize');
 module.exports = (sequelize, DataTypes) => {
   class GrantCycle extends Model {
     static associate(models) {
       GrantCycle.associate = function (models) {
         GrantCycle.hasMany(models.Nomination, {
-          foreignKey: 'grantCycleId',
+          foreignKey: 'nominationId',
```

Comment:
This was actually correct before.

The `foreignKey` refers to the key that the target model (the Nomination model in this case) uses to associate itself with the current GrantCycle model. In other words, the Nomination table in the database needs have a column named "grantCycleId" so it can know which GranCycle each row belongs too.

Also, looking at the [Sequelize docs](https://sequelize.org/master/manual/assocs.html), it seems that the `foreignKey` is automatically added an the appropriate table. So we can just remove this line!